frontend/src/app/animals-list/animals-list.component.ts

```diff
@@ -11,18 +13,33 @@ import { Observable } from 'rxjs';
 })
 export class AnimalsListComponent implements OnInit {
   public animals$: Observable<Animal[]>;
-  countResult: number;
+  public countResult: number;
+  public animalClassTitle = 'animals';
```

Comment:
Why do you need `animalClassTitle` here? Why do you can't insert it directly into template?